### DISCUSSION TITLE
Where can we get some Github Free Certifications

### DISCUSSION THREAD

**Discussion by HAQ-NAWAZ-MALIK** [community/community#110991] [Other Feature Feedback, Questions, & Ideas] [answered]
Where can we get some Github Free Certifications??

**Comment by HAQ-NAWAZ-MALIK** [comment]
anyone help ?

**Reply by BrendaThistle** [reply]
@HAQ-NAWAZ-MALIK send us your email address (gh-certification@github.com). We may have a couple vouchers left over from the education team.

**Reply by HAQ-NAWAZ-MALIK** [reply]
I am so happy and thankful for your response, i sent my email to your said email address ( gh-certification@github.com ).

Btw,

Do i need to pay for certifications. Or can I get free by student dev pack? Or any way

**Comment by queenofcorgis** [comment]
Welcome to the GitHub Community, @HAQ-NAWAZ-MALIK , we're happy you're here! You are more likely to get a useful response if you are posting your question in the applicable category and are explicit about what it entails--giving a few more details might help someone give you a nudge in the right direction. I've gone ahead and moved it for you. Good luck!

**Reply by HAQ-NAWAZ-MALIK** [reply]
Thank you , I will do the same.

**Comment by SebastienXia** [comment]
The exams are taken in an interview or on a sort of MCQ ?

**Comment by BrendaThistle** [comment]
Hey there! Our [website](https://resources.github.com/learn/certifications/), [Candidate Handbook](https://examregistration.github.com/handbook) and [FAQ](https://examregistration.github.com/faq) have a lot of useful information. The exams are currently proctored through PSI. You can do them in person or online. You can register for any of the exams through our [website](https://resources.github.com/learn/certifications/).

**Comment by moondoor01** [comment]
Could you tell me what Github Free Certifications are?

**Reply by SirBang** [reply]
With GitHub Certifications, you can showcase your expertise in GitHub technologies and workflows. Earning a GitHub Certification gives you a competitive edge in the job market by allowing you to promote your skills in a specific GitHub domain.

**Comment by SirBang** [comment]
What kind of Certs do you want to get?

**Reply by HAQ-NAWAZ-MALIK** [reply]
Are these paid Certification, ? 
Or free?.

**Comment by samus-aran** [comment, marked as answer]
Hi everyone!

Thank you for your interest in [GitHub Certifications](https://resources.github.com/learn/certifications/). We do have a limited number of vouchers for verified GitHub students and teachers to specific GitHub Certifications for free through to June 30th. If you wish to apply please email us on [gh-certification@github.com](mailto:gh-certification@github.com) and the GitHub Certifications team will review your request.
